series/praha-pyvo/events/2017-06-14-after-pycon.yaml

```diff
@@ -30,6 +31,7 @@ talks:
   speakers:
   - Petr Hodač
   urls: []
-  coverage: []
+  coverage:
+  - video: https://www.youtube.com/watch?v=nYcZebvI8aI
```

Comment:
Tohle video je rozbité v čase 17:0něco je tam půlsekundová pauza v audiu a dál je to nesynclé. Ještě jsem se na to nestihl podívat a opravit to :(

Reply:
Nastavil jsem ho na Unlisted.
Je jediné takové?

Reply:
Jsou to všechny, které měly víc než jedno zdrojové video z kamery, tedy všechy delší než 17 minut. Včera jsem to předělal tak, aby se na ta videa použil vstupní filtr concat místo spojování jako samostatných videosouborů a to už funguje dobře, dnes vyzkoušeno - viz encukou/talk-video-maker#6

Reply:
Aj!
OK, všechny delší než 12 minut jsem nastavil jako unlisted. Dáš mi prosím vědět, až budou k dispozici spravené verze? Nebo je mám na YT dávat postupně?

Reply:
Nebo ti dám přístup na YouTube, když mi pošleš e-mailovou adresu s Google účtem

Reply:
Já je klidně budu na YT nahrávat (ideálně automaticky skriptem jako unlisted a pak je ručně zveřejníme když budou dobrá). Můj google účet je jmeno.prijmeni at gmail

Reply:
Přidal jsem tě.
Poprosím jen o zveřejnění skriptů, pokud to bude možné, i kdyby byly škaredé a fungovaly jen na tvém počítači :)

Reply:
Tak jsem prošel adresář done a všechny rozbitá videa přesunul do ../broken. Pustím na ně maker znovu a měla by vypadnout už opravená videa. Na nahrávací skript se pustím asi časem. Rozhodně ho chci zveřejnit.